source/CardGames.py

```diff
@@ -284,9 +284,25 @@ def character_selection(self):
       return Player1, Player1Role, Player2, Player2Role
 
   
-    # Filler function until we implement the function to check if the guess is correct
     def check_location(self, guess, marshall_current_idx):
-      return True
+      # Loop if guessing multiple locations
+      if type(guess) == list:
+        for i in range(marshall_current_idx, len(self.cards_in_play)):
+          guess_idx = i - marshall_current_idx
+          item = self.cards_in_play[i]
```

Comment:
Nice job, Sean! Are we storing every card within the list, cards_in_play, as either a list of Card objects or a singular Card object?

Reply:
Yes